src/lumps/doom/mapThingType.ts

```diff
@@ -178,3 +178,13 @@ export const WADMapThingTypeList: WADMapThingType[] = [
     {id: 0x057, sprite: "", radius: 0, name: "Spawn spot", class: WADMapThingClass.Spawner},
     {id: 0x00E, sprite: "", radius: 20, name: "Teleport landing", class: WADMapThingClass.Teleport},
 ];
+
+// Similar to the list above, but uses the thing IDs as the keys.
+// Getting values from a hash map (Object) is faster than iterating through an array.
+export const WADMapThingTypeMap: {[type: number]: WADMapThingType} = (() => {
```

Comment:
Good addition, though a name such as `WADMapThingTypeById` or `WADMapThingTypeByIdMap` would be my preference, in order to distinguish it from any possible future mapping strategies